pkg/eks/nodegroup.go

```diff
@@ -55,6 +55,41 @@ func (c *ClusterProvider) CreateNodeGroupAuthConfigMap(clientSet *clientset.Clie
 	return nil
 }
 
+// AddNodeGroupToAuthConfigMap updates the auth config map to include the node group
+func (c *ClusterProvider) AddNodeGroupToAuthConfigMap(clientSet *clientset.Clientset, ng *api.NodeGroup) error {
+	cm, err := clientSet.CoreV1().ConfigMaps("kube-system").Get("aws-auth", metav1.GetOptions{})
+	if err != nil {
+		return errors.Wrapf(err, "failed getting auth ConfigMap for %s", ng.Name)
```

Comment:
We should allow the case where cluster was created without a nodegroup, and thereby config map never been created also. It's not allowed via CLI right now, but we would want to allow it soon.

Reply:
https://github.com/weaveworks/eksctl/issues/376